assets/js/index.js

```diff
@@ -0,0 +1,296 @@
+const API_KEY = "7ec1ea2463d21d115915eb7b42565bed";
+
+const searchHistoryContainer = $("#search-history-container");
+
+const searchBtn = $("#search-button");
+
+const form = $("#select-city");
+
+const formInput = $("#input-city");
+
+const searchHistory = $("#search-history");
+
+const cityButton = $("#city-button");
+
+const weatherContainer = $("#weather-container");
+
+const currentWeather = $(".current-weather");
+
+const forecastContainer = $("#forecast-container");
+
+const currentDate = moment.unix().format("dddd Do MMMM YYYY");
+
+// initialise LS
+const onReady = () => {
+  // check if cities exists in LS
+  const recentCities = readFromLocalStorage();
+  // if false then set tasks to empty object in LS
+  if (!recentCities) {
+    localStorage.setItem("cities", JSON.stringify([]));
+  }
+
+  renderCities();
+};
+
+// get cities from local storage
+const readFromLocalStorage = () => {
```

Comment:
You could make this function more reusable by passing in the key, rather than hard coding. It's not really necessary in this particular homework, but imagine you wanted to get different things from local storage through your app, you could do this:

```
const readFromLocalStorage = ( key ) => {
  const getCities = localStorage.getItem(key);
  return JSON.parse(getCities);
```

And then when you call it, you just pass in the key you want at that time:

`readFromLocalStorage("cities")`